pom.xml

```diff
@@ -972,7 +968,7 @@
                                         <execute>
                                             <runOnIncremental>false</runOnIncremental>
                                         </execute>
-                                        <ignore />
+                                        <ignore></ignore>
```

Comment:
Right, there is a **<expandEmptyElements>** set by default to **true**. I will fix this.